qgreenland/config/datasets/asiaq_nunagis.py

```diff
@@ -15,6 +15,16 @@
             output_file="{output_dir}/fetched.geojson",
             url="https://kort.nunagis.gl/refserver/rest/services/Kortportal/Kortportal_TekniskGrundkort/MapServer/38/query/?f=json&where=OBJECTID+is+not+null&outFields=*&orderByFields=OBJECTID+ASC",
         ),
+        ogr_remote_asset(
+            asset_id="settlements",
+            output_file="{output_dir}/fetched.geojson",
+            url="https://kort.nunagis.gl/refserver/rest/services/Kortportal/Byer_og_bygder/MapServer/0/query/?f=json&where=OBJECTID+is+not+null&outFields=*&orderByFields=OBJECTID+ASC",
```

Comment:
This is actually a different service than the other assets. We should either:

* Create a new dataset for this service/layers
* Edit the existing dataset metadata to indicate NunaGIS more broadly.

I reached out to our contacts at Asiaq earlier today and requested their input on how to cite the data layers we are adding. If they indicate a citation for NunaGIS/Asiaq more boradly, then we can go with option two. If we need citations specific to each service (or layer), then we may need to go with option 1.

Reply:
We decided to go with option 2.